index.bs

```diff
@@ -297,8 +297,28 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
       then return true.</li>
       <li>Otherwise, for each <var>item</var> in the <a>allowlist</a>:
         <ol>
-          <li>If <var>item</var> is [=same origin-domain=] with
-          <var>origin</var>, then return true.</li>
+          <li>If <var>item</var> is a <a>serialized-origin</a> that's [=same origin-domain=] 
+          with <var>origin</var>, then return true.</li>
+          <li>Else if <var>item</var> is a <a>serialized-origin-with-wildcard-subdomain</a>
+          then:</li>
+          <ol>
+            <li>If <var>origin</var> does not have a host with a <a>registrable domain</a>
```

Comment:
Can you link "host" as well? "does not have" is also a bit ambigious, and it might be better to link origin to https://html.spec.whatwg.org/multipage/origin.html#concept-origin and indicate that the host is not part of the tuple.

Reply:
Also, the registrable domain algorithm can return null, but there's no concept of a host without a registrable domain..